msal/src/Microsoft.Identity.Client/IPublicClientApplication.cs

```diff
@@ -288,5 +288,20 @@ public partial interface IPublicClientApplication : IClientApplicationBase
         // endif !NOT_CORE
 #endif
 
+#if MSAL_FEATURE_USERNAMEPASSWORD
```

Comment:
do we have to say `MSAL` can it just be `FEATURE`?

Reply:
I like MSAL_FEATURE because if we have to add it to core files, it will clearly disambiguate between an ADAL and an MSAL feature